File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0113U00GrdOCS0113ListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0113U00GrdOCS0113ListItemInfo
	{
		private String _hangmogCode;
		private String _seq;
		private String _defaultYn;
		private String _specimenCode;
		private String _specimenName;
		private String _hangmogStatDate;
		private String _rowState;

		public String HangmogCode
		{
			get { return this._hangmogCode; }
			set { this._hangmogCode = value; }
		}

		public String Seq
		{
			get { return this._seq; }
			set { this._seq = value; }
		}

		public String DefaultYn
		{
			get { return this._defaultYn; }
			set { this._defaultYn = value; }
		}

		public String SpecimenCode
		{
			get { return this._specimenCode; }
			set { this._specimenCode = value; }
		}

		public String SpecimenName
		{
			get { return this._specimenName; }
			set { this._specimenName = value; }
		}

		public String HangmogStatDate
		{
			get { return this._hangmogStatDate; }
			set { this._hangmogStatDate = value; }
		}

		public String RowState
		{
			get { return this._rowState; }
			set { this._rowState = value; }
		}

		public OCS0113U00GrdOCS0113ListItemInfo() { }

		public OCS0113U00GrdOCS0113ListItemInfo(String hangmogCode, String seq, String defaultYn, String specimenCode, String specimenName, String hangmogStatDate, String rowState)
		{
			this._hangmogCode = hangmogCode;
			this._seq = seq;
			this._defaultYn = defaultYn;
			this._specimenCode = specimenCode;
			this._specimenName = specimenName;
			this._hangmogStatDate = hangmogStatDate;
			this._rowState = rowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0113U00GrdOCS0113ListItemInfo.java
package nta.med.data.model.ihis.ocsa;

import java.util.Date;

public class OCS0113U00GrdOCS0113ListItemInfo {
	private String hangmogCode;
	private Double seq;
	private String defaultYn;
	private String specimenCode;
	private String specimenName;
	private Date hangmogStatDate;
	private String rowState;

	public OCS0113U00GrdOCS0113ListItemInfo(String hangmogCode, Double seq,
			String defaultYn, String specimenCode, String specimenName,
			Date hangmogStatDate, String rowState) {
		super();
		this.hangmogCode = hangmogCode;
		this.seq = seq;
		this.defaultYn = defaultYn;
		this.specimenCode = specimenCode;
		this.specimenName = specimenName;
		this.hangmogStatDate = hangmogStatDate;
		this.rowState = rowState;
	}

	public String getHangmogCode() {
		return hangmogCode;
	}

	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}

	public Double getSeq() {
		return seq;
	}

	public void setSeq(Double seq) {
		this.seq = seq;
	}

	public String getDefaultYn() {
		return defaultYn;
	}

	public void setDefaultYn(String defaultYn) {
		this.defaultYn = defaultYn;
	}

	public String getSpecimenCode() {
		return specimenCode;
	}

	public void setSpecimenCode(String specimenCode) {
		this.specimenCode = specimenCode;
	}

	public String getSpecimenName() {
		return specimenName;
	}

	public void setSpecimenName(String specimenName) {
		this.specimenName = specimenName;
	}

	public Date getHangmogStatDate() {
		return hangmogStatDate;
	}

	public void setHangmogStatDate(Date hangmogStatDate) {
		this.hangmogStatDate = hangmogStatDate;
	}

	public String getRowState() {
		return rowState;
	}

	public void setRowState(String rowState) {
		this.rowState = rowState;
	}

}
